manifests/init.pp

```diff
@@ -96,6 +98,7 @@
   Boolean $service_ensure = true,
   Optional[Boolean] $service_enable = undef,
   Enum['present', 'absent'] $ensure = 'present',
+  Stdlib::AbsolutePath $data_dir = '/var/lib/qpidd/.qpidd',
```

Comment:
The default data directory in qpid-cpp is this path here. This implementation would manage this directory but would not manage the top level directory `/var/lib/qpidd` (which is created by the qpid-cpp RPM).

Options:

 1) Stick with this pattern as that is how things work today
 2) Switch to specifying `/var/lib/qpidd` as the data directory so we can manage and remove it entirely as a single entity. This should not break upgrades given the configuration  would get updated and restart the qpidd service. Any old `.qpidd` data would then get cleaned up anytime that someone set qpid to ensure absent,

Reply:
I'm in favor of using `/var/lib/qpidd`, mostly because that's the home:
```console
# getent passwd qpidd
qpidd:x:998:998:Owner of Qpidd Daemons:/var/lib/qpidd:/sbin/nologin
# rpm -qv qpid-cpp-server-linearstore       
qpid-cpp-server-linearstore-1.39.0-1.el7.x86_64
```

So I'd name it `user_home` and remove that.

Reply:
I made the change to `/var/lib/qpidd` but stuck with `data_dir` as the parameter since that is the Qpid configuration that is being specified.